sw/host/opentitanlib/src/util/usr_access.rs

```diff
@@ -35,3 +66,34 @@ pub fn usr_access_get(bitstream: &[u8]) -> Result<u32> {
     Ok(usr_access)
 }
 
+/// Returns a 32-bit timestamp suitable to be used as a USR_ACCESS value:
+///
+/// |-------+-------+-------+-------------------------+-------+--------+--------|
+/// | Bits: | 31:27 | 26:23 | 22:17                   | 16:12 | 11:6   | 5:0    |
+/// |-------+-------+-------+-------------------------+-------+--------+--------|
+/// | Data: | Day   | Month | Year (since 2000, 0-63) | Hour  | Minute | Second |
+/// |-------+-------+-------+-------------------------+-------+--------+--------|
+///
+/// From
+/// https://www.xilinx.com/content/dam/xilinx/support/documents/application_notes/xapp1232-bitstream-id-with-usr_access.pdf
+pub fn usr_access_timestamp() -> u32 {
```

Comment:
For testing purposes, it might make more sense take a `DateTime<Utc>` parameter.

Reply:
Yeah, this comment is only relevant if you're going to write a unit test for the function. It doesn't seem painful to refactor later either, so I don't feel strongly :)